File: LES_EXAMENS/EXAMEN_PYTHON_2025_2026_IA/exercice6.py
S=0
i=0
while True:
    i+=1
    prix=float(input(f"saisir le prix numero {i}: "))
    if prix!=0:
        S=S+prix
    else:
        print("Merci d'utiliser ce logiciel.")
        break
M=S/(i-1)
print(f"la somme est {S}.")
print(f"le moyenne est {M} .")

File: LES_EXAMENS/EXAMEN_PYTHON_2025_2026_IA/exercice6.c
#include <stdio.h>
float M,S=0,prix;
int i=0;
int main(){
    do{
        i++;
        printf("saisir le prix numero {%d}: ",i);
        scanf("%f",&prix);
        if(prix!=0)
            S=S+prix;
        else
            printf("Merci d'utiliser ce logiciel.\n");
    }while(prix!=0);
    M=S/(i-1);
    printf("la somme est %f .\n",S);
    printf("le moyenne est %f .\n",M);
    return 0;
}